drivers/web-worker-driver/karma.config.d/wasm.js

```diff
@@ -25,3 +25,6 @@ config.files.push({
   watched: false,
   included: false,
 });
+
+// TODO: Figure out why on earth this is necessary. Presumably a karma-webpack bug???
```

Comment:
Hi, I am Kotlin Gradle plugin developer, and currently working on support of JS and Wasm part in Gradle plugin.
I can add some piece of information why this code is useful in cases of wasm using.

When webpack making bundle, it process dynamic imports to separate files, to before webpack work, you can't know how many files webpack is going to create. It is bad for Karma, it wants to know exactly how many files it should serve for testing.
And karma-webpack plugin makes a bit hacky solution, it tells webpack to pack all modules (async and non-async) to one big bundle. It works well for Karma, but there is problem with sourcemaps in this case! For Kotlin source maps are crucially important, so we initially disabled this behaviour for Karma via `webpack.optimization`, we did it in Kotlin/JS because we have a small count of async modules there, and this combination of settings worked well.
But with wasm, where there are a lot of async modules, it stopped to work. That's why this workaround can be redundant in wasm cases.

But.. I can see another workaround here with webpack `output`. So theoretically it means that Karma has to see all files which webpack has emitted, so you need either workaround with `webpack.output` or with `webpack.optimization. Theoreticlly you don't need both.

But of course I can be wrong here, maybe you have some specific use case which I don't know about.